prototype/build/node.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM golang:1.17.6-bullseye
```

Comment:
Why do we need a different Dockerfile for this? Ideally a single Dockerfile should be sufficient for all purposes.

Again, seems like the PR is not ready for submission?

Reply:
1- This is the node's p2p docker image that is used to the e2e test. Should we use the other dockerfile instead? Not sure, but historically I think this precedes the other files that's why it's stood there.

2- Do you think so? I think the review is doing its job so no worries as functionality wise it's what I want to submit at this point, so that future work is streamlined and this does not lag behind and cause me conflict headache.